src/components/Pattern/Pattern.styles.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+export const PatternWrapper = styled.div`
+  /* padding: var(--small-space);
+  border: 0.1rem solid var(--logopink); */
+  .lightpink {
+    fill: var(--lightpink);
+  }
+  .darkpink {
+    fill: var(--darkpink);
+  }
+  .mattepink {
+    fill: var(--mattepink);
+  }
+  .logopink {
+    fill: var(--logopink);
+  }
+  .lightgrey {
+    fill: var(--lightgrey);
+  }
+  .darkgrey {
+    fill: var(--darkgrey);
+  }
+  .darkburgundy {
+    fill: var(--darkburgundy);
+  }
+  .orange {
+    fill: var(--orange);
+  }
+  .lightblue {
+    fill: var(--lightblue);
+  }
+  .midblue {
+    fill: var(--midblue);
+  }
+  .darkblue {
+    fill: var(--darkblue);
```

Comment:
Once you pass `color` in to PatternWrapper you can get rid of all these classes and just add:

```
export const PatternWrapper = styled.div`
   svg {
       fill: ${({color}) => color};
   }   
`;
```